.github/workflows/continuous-monitoring.yml

```diff
@@ -0,0 +1,43 @@
+name: Continuous monitoring of distribution channels
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 */1 * * *'
+
+jobs:
+  pull-agent:
+    name: Pull X-Ray java agent from Maven Central and Github
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+                 
+      - name: Pull Java agent from Github
+        id: distribution-availability-github
+        run: wget https://github.com/aws/aws-xray-java-agent/releases/download/v2.8.0/xray-agent.zip
+         
+      - name: Publish metric on X-Ray Java agent distribution availability (Github)
+        if: ${{ always() }}
+        run: |
+          if [[ "${{ steps.distribution-availability-github.outcome }}" == "failure" ]]; then
+            aws cloudwatch put-metric-data --metric-name XRayJavaAgentGithubDistributionUnavailability --dimensions failure=rate --namespace MonitorAgent --value 1 --timestamp $(date +%s)
+          else
+            aws cloudwatch put-metric-data --metric-name XRayJavaAgentGithubDistributionUnavailability --dimensions failure=rate --namespace MonitorAgent --value 0 --timestamp $(date +%s)
+          fi
+          
+      - name: Pull Java agent from Maven
+        id: distribution-availability-maven
+        run: wget https://search.maven.org/artifact/com/amazonaws/aws-xray-agent-plugin/2.8.0/aws-xray-agent-plugin-2.8.0.jar
```

Comment:
Similarly here, maybe to avoid manual updates on each new release, we can get the latest tag on the repo and use that to derive the URL instead of hardcoding a version number.

Reply:
Also, we should probably use the direct maven repository URL: https://repo.maven.apache.org/maven2/com/amazonaws/aws-xray-agent-plugin/2.8.0/aws-xray-agent-plugin-2.8.0.jar

This one will have much less latency to propagate for new releases, so there won't be false failures in the metrics immediately after a release.